src/renderer/containers/ReplayFileStats/HomeRunProfile.tsx

```diff
@@ -0,0 +1,26 @@
+import Warning from "@mui/icons-material/Warning";
+import type { FileResult } from "@replays/types";
+import type { StatsType } from "@slippi/slippi-js";
+import _ from "lodash";
+import React from "react";
+
+import { IconMessage } from "@/components/Message";
+
+export interface GameProfileProps {
+  file: FileResult;
+  stats: StatsType;
+}
+
+const StatSection: React.FC<{
+  title: string;
+}> = () => {
+  return <IconMessage Icon={Warning} label="🛠 This page is under construction" />;
```

Comment:
<img width="344" alt="Screen Shot 2023-01-30 at 11 33 03" src="https://user-images.githubusercontent.com/8384734/215375339-258d1187-9f7d-49c2-8023-21e4f25f5019.png">

I think instead of having an emoji and an icon, you can just replace the `Warning` icon with one of these.